.github/workflows/cuda.yml

```diff
@@ -133,17 +129,17 @@ jobs:
         ccache -s
         du -hs ~/.cache/ccache
 
-  build_nvhpc24-1-nvcc:
-    name: NVHPC@24.1 NVCC/NVC++ Release [tests]
-    runs-on: ubuntu-20.04
+  build_nvhpc:
+    name: NVHPC
+    runs-on: ubuntu-24.04
```

Comment:
No need to change, because we use on PM now another env:

But long term, for NVHPC I would also use an "older" release to cover what Cray/HPE deploy usually.